.github/workflows/optimize-generate-cambpm-test-datasets.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     concurrency:
       cancel-in-progress: true
       group: ${{ github.workflow }}-${{ inputs.sql-dump-filename }}
-    runs-on: gcp-core-32-longrunning-big-ssd
+    runs-on: gcp-core-32-longrunning
```

Comment:
Is there a reason why we did not add the gcp-core-32-longrunning-big-ssd to the list in `.github/actionlint.yaml` instead?

Reply:
Worth nothing - I think we should simply delete this workflow anyway, and also any workflows that use the data created by it. It's for generating C7, so serves no real place in the mono repo. If we want an equivalent workflow for C8, I think it would be better to build from scratch or history, rather than trying to modify this